pkg/virt-api/validating-webhook/validating-webhook.go

```diff
@@ -321,6 +321,17 @@ func validateVirtualMachineSpec(field *k8sfield.Path, spec *v1.VirtualMachineSpe
 		return causes
 	}
 
+	// Validate requested memory size if value is not negative
+	if spec.Domain.Resources.Requests.Memory().Value() < 0 {
```

Comment:
Can you do the same for spec.Domain.Resources.Limits.Memory().Value()? Also Please add a unit test.

Reply:
sure :)

Reply:
Done